model/src/main/kotlin/OrtResult.kt

```diff
@@ -463,4 +463,9 @@ data class OrtResult(
                 result = analyzer.result.withScopesResolved()
             )
         )
+
+    /**
+     * Return an object that can be used to navigate the dependency information contained in this result.
+     */
+    fun dependencyNavigator(): DependencyNavigator = DependencyTreeNavigator
```

Comment:
Why is this hard-coded to a "Dependency*Tree*Navigator"? Is it because the *tree*, as opposed to the *graph*, is available for all package manager implementations?